### DISCUSSION TITLE
Remove file added notification when bulk uploading files from the UI

### DISCUSSION THREAD

**Discussion by vmoriaux** [immich-app/immich#10615] [General]
I'm manually importing some files as my wife doesn't want to use Immich. I'm not using an external library, just the default lib with storage template.

When uploading several files, the import progress is hidden behind the "file added" notifications
![image](https://github.com/immich-app/immich/assets/1343895/e9e99c05-6fb4-491a-9e75-2e375e63e016)

I don't find these notifications really useful in this context since I'm just dropping all the files in the same album. And I can't really see the batch upload panel at the bottom right that summarize the number of duplicates and/or failure.
So my suggestion is to remove the notifications or combine them into a single one at the top of the screen saying "X asset added to the album" and make sure the import panel is visible.

It would be even more helpful if this panel would stay until manually closed, so we could imagine be able to
- click on the failed import count to see the error messages
- or click on the duplicate count to see the list and maybe do some bulk action on them.

**Comment by bo0tzz** [comment]
#6102
